src/udSDK/udSDKProject.py

```diff
@@ -252,7 +252,7 @@ def nextSibling(self):
         The next node at the same level in the tree, None if this is the last node in this level
         """
         if self.parent is not None:
-            return self.parent.child_from_pointer(self.pNextSibling)
+            return self.from_pointer(self.pNextSibling)
         else:
             return None
```

Comment:
This doesn't need to do the check for `self.parent` anymore, this should work the same as `firstChild`.

Reply:
Worse- due to a botched renaming child_from_pointer doesn't exist any more 
I'll address this in a separate PR

Reply:
It's not using `child_from_pointer`, it's using `from_pointer`?

Reply:
My bad- misread what was going on - fixed in #48